integration_tests/publish_intent_requests_test.go

```diff
@@ -0,0 +1,153 @@
+package integration
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/alphagov/publishing-api"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/ghttp"
+)
+
+var _ = Describe("Publish Intent Requests", func() {
+	var (
+		contentItemJSON = `{
```

Comment:
This isn't really a `contentItemJSON`, it's a publishIntentJSON. The content should really change too to be like: https://github.com/alphagov/content-store/blob/master/doc/publish_intents.md#publish-intent-format